### DISCUSSION TITLE
GitHub Discussions Open

### DISCUSSION THREAD

**Discussion by jeremyevans** [jeremyevans/sequel#1792] [General]
Previously, Sequel used the sequel-talk Google Group as a discussion forum/mailing list.  That will continue to be available for people who do not want to use GitHub Discussions, but GitHub Discussions is now the preferred discussion forum for Sequel.  Feel free to ask any questions regarding Sequel here.  Bugs in Sequel should still be reported via GitHub Issues.

**Comment by tycooon** [comment]
This is great news and thanks for your work!

**Comment by hmdne** [comment]
This is great news for those who can't have, don't have or refuse to have a Google account!
